packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/en.ftl

```diff
@@ -9,11 +9,5 @@ complete-reset-password-success-alert = Password set
 # Displayed in an alert bar
 complete-reset-password-error-alert = Sorry, there was a problem setting your password
 
-password-reset-data-may-not-be-recovered = Resetting your password may delete your encrypted browser data.
-password-reset-could-not-determine-account-recovery-key = Have an account recovery key?
-password-reset-use-account-recovery-key = Reset your password with your recovery key.
-password-reset-previously-signed-in-device = Have a device where you previously signed in?
-password-reset-data-may-be-saved-locally = Your browser data may be locally saved on that device. Sign in there with your new password to restore and sync.
-password-reset-no-old-device = Have a new device but don’t have your old one?
-password-reset-encrypted-data-cannot-be-recovered = We’re sorry, but your encrypted browser data on Firefox servers can’t be recovered.
-password-reset-learn-about-restoring-account-data = Learn more about restoring account data.
+password-reset-could-not-determine-account-recovery-key = Got your account recovery key?
+password-reset-use-account-recovery-key-2 = Reset your password and keep your data
```

Comment:
nit: I don't think this string has been translated yet, so the -2 would be necessary.